src/arrayMethodSort.js

```diff
@@ -1,11 +1,44 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+function compareFormstring(first, second) {
+  let a = first;
+  let b = second;
+
+  a = String(a);
+  b = String(b);
```

Comment:
You can actually combine these 4 lines into 2.

```suggestion
  const a = String(first);
  const b = String(second);
```